src/routes/legislative-process/+page.ts

```diff
@@ -0,0 +1,143 @@
+		}
+	];
+	const senates: DutySection[] = [
+		{
+			heading: 'พิจารณาร่างกฎหมาย (พ.ร.บ.)',
+			details: [
+				'กลั่นกลองและให้ความเห็นร่างกฎหมายที่ สส. เห็นชอบ',
+				'ยับยั้งร่างกฎหมายเพื่อส่งให้ สส. พิจารณาอีกครั้ง'
+			]
+		},
+		{
+			heading: 'อนุมัติ พ.ร.ก. หรือกฎหมายฉุกเฉินที่ออกโดย ครม. ให้เป็นกฎหมาย',
+			details: [
+				'ลงมติเห็นชอบหรือไม่เห็นชอบ พ.ร.ก.',
+				'เสนอต่อประธานสภาฯ เพื่อให้ประธานสภาฯ ส่งเรื่องไปให้ศาลรัฐธรรมนูญวินิจฉัย และให้ชะลอ พ.ร.ก.ไว้ก่อน'
+			]
+		},
+		{
+			heading: 'ควบคุมการบริหารราชการแผ่นดิน',
+			details: [
+				'ตั้งกระทู้ถามรัฐมนตรีเกี่ยวกับงานในหน้าที่ในรูปแบบเป็นหนังสือหรือด้วยวาจา และสามารถตั้งกระทู้ถามด้วยวาจาโดยไม่ต้องแจ้งล่วงหน้า',
+				'ตรวจสอบและเร่งรัดให้รัฐบาลดำเนินงานตามแผนยุทธศาสตร์ชาติ',
+				'ยื่นให้ศาลรัฐธรรมนูญวินิจฉัยชี้ขาดกรณีรัฐบาลไม่ดำเนินงานตามยุทธศาสตร์ชาติ'
+			]
+		},
+		{
+			heading: 'ดำเนินกิจการสภา',
+			details: [
+				'เลือก สว. เป็นกรรมาธิการคณะกรรมาธิการสามัญ',
+				'เลือก สว.หรือผู้ที่ไม่ได้เป็น สว. ตั้งเป็นคณะกรรมาธิการวิสามัญ เพื่อศึกษาเรื่องใดๆ และรายงานให้สภาทราบตามระยะเวลาที่สภากําหนด'
+			]
+		},
+		{
+			heading: 'แต่งตั้งผู้ดำรงตำแหน่งในองค์กรตามรัฐธรรมนูญ',
+			details: [
+				'ลงมติเห็นชอบผู้ที่ดำรงตำแหน่งตุลาการศาลรัฐธรรมนูญ',
+				'พิจารณาแต่งตั้งบุคคลในองค์กรอิสระ ได้แก่ ป.ป.ช., ก.ก.ต., ผู้ตรวจการแผ่นดิน, คณะกรรมการผู้ตรวจเงินแผ่นดิน'
+			]
+		}
+	];
+
+	const both: DutySection[] = [
+		{
+			heading: 'ด้านนิติบัญญัติ (การตรากฎหมาย)',
+			details: [
+				'พิจารณาร่างพระราชบัญญัติประกอบรัฐธรรมนูญ (พ.ร.ป.)',
+				'ปรึกษาร่าง พ.ร.ป. หรือร่าง พ.ร.บ.ใหม่',
+				'พิจารณาให้ความเห็นชอบตามที่ ครม. ร้องขอ',
+				'ตราข้อบังคับการประชุมรัฐสภา',
+				'แก้ไขเพิ่มเติมรัฐธรรมนูญ'
+			]
+		},
+		{
+			heading: 'ด้านการบริหารราชการแผ่นดิน',
+			details: [
+				'ประชุมร่วมกันของรัฐสภา',
+				'รับฟังคำแถลงนโยบายของ ครม.',
+				'เปิดอภิปรายทั่วไปในที่ประชุมร่วมกันของรัฐสภา'
+			]
+		},
+		{
+			heading: 'ด้านการประชุมรัฐสภา',
+			details: [
+				'เปิดประชุมรัฐสภาครั้งแรก',
+				'ให้ความเห็นชอบในการปิดสมัยประชุม',
+				'ตราข้อบังคับการประชุมรัฐสภา'
+			]
+		},
+		{
+			heading: 'ด้านการให้ความเห็นชอบ',
+			details: [
+				'ให้ความเห็นชอบในการประกาศสงคราม',
+				'รับฟังคำชี้แจงและให้ความเห็นชอบหนังสือสัญญาที่มีบทเปลี่ยนแปลงอาณาเขต อำนาจอธิปไตย หรือกระทบต่อความมั่นคงทางเศรษฐกิจ สังคม หรือการค้าของประเทศอย่างกว้างขวาง'
+			]
+		},
+		{
+			heading: 'กรณีตามบทเฉพาะกาล',
+			details: [
+				'ลงมติเลือกนายกรัฐมนตรี ตามมาตรา 272 (5 ปีแรกของรัฐสภาชุดแรก)',
+				'พิจารณาร่าง พ.ร.บ. ที่ตราขึ้นตามหมวด 16 การปฏิรูปประเทศ',
+				'พิจารณาร่าง พ.ร.บ. ที่ สว. หรือ สส. ยับยั้งไว้ตามมาตรา 271'
+			]
+		},
+		{
+			heading: 'ด้านหมวดพระมหากษัตริย์',
+			details: [
+				'ให้ความเห็นชอบในการแต่งตั้งผู้สำเร็จราชการแทนพระองค์',
+				'รับทราบการปฏิญาณตนของผู้สำเร็จราชการแทนพระองค์ต่อรัฐสภา',
+				'รับทราบการแก้ไขเพิ่มเติมกฎมณเฑียรบาลว่าด้วยการสืบราชสันตติวงศ์พระพุทธศักราช 2467',
+				'รับทราบหรือให้ความเห็นชอบในการสืบราชสมบัติ'
+			]
+		}
+	];
+
+	return {
+		representatives,
+		senates,
+		both
```

Comment:
I suggest you put these 3 variables under `dutySection` property so that the other (dev who works on the other) section will understand that these variables are not related to them.